### ISSUE TITLE
Fingerprint cannot be registered (Kebab)

### ISSUE THREAD

**Issue by plaggeds** [PixelExperience/android-issues#2432]
## Build date

PixelExperience_Plus_kebab-12.1-20220410-0820-OFFICIAL

## Expected Behavior

When registering fingerprint, animation for registering should appear

## Current Behavior

When pressing on fingerprint sensor, the portion of the screen lights up but the registration animation is not shown

## Possible Solution

N/A

## Steps to Reproduce

1. Set Up Fingerprint
2. Press on the fingerprint sensor

/device kebab
/version twelve_plus

**Comment by pixelexperiencebot** [comment]
Issue created! You can close at any time by commenting ```/close```

**Comment by plaggeds** [comment]
https://drive.google.com/file/d/1-27GtxrhGsQP9oJNqgLx_z4oZyCwH3EA/view?usp=drivesdk

Here is a video example of the issue

**Comment by Shadowrazej** [comment]
+1 have the same issue.
I absolutely cannot add fingerprint.
PixelExperience_kebab-12.1-20220410-0653-OFFICIAL.zip

Reinstalled ROM 2-3 times.

**Comment by Shadowrazej** [comment]
Hi man.
How I solved this issue. I hope it will works correctly.
I rolled back to stock ROM (Oxygen 11) via MSMTool. Made OTA update (Settings -> updates) to latest OxygenOS 11,  OTA update to OxygeonOS 12.
Configured in OxygenOS 12 at least 1 fingerprint.

Only now I install PixelExperience Android 12. And now I can add fingerprints.


I think that the main problem, that I wanted to install PixelExperience Android 12 ROM over stock OxygenOs 11.


One more way to solve this problem: 
To install magisk and some modules. But I don't really like it. 
Link to the instruction - https://4pda.to/forum/index.php?showtopic=1009674&st=3040#entry114450245
